news/rs.rst

```diff
@@ -0,0 +1,42 @@
+**Added:**
+
+* New subprocess specification class ``SubprocSpec`` is sued for specifiying
+  and manipulating subprocess classes prior to execution.
+* New ``PopenThread`` class runs subprocesses on a a separate thread.
+* New ``CommandPipeline`` and ``HiddenCommandPipeline`` classes manage the
+  execution of a pipeline of commands via the execution of the last command
+  in the pipeline. Instances may be iterated and stream lines from the
+  stdout buffer.
+* ``$XONSH_STORE_STDOUT`` is now available on all platforms!
+* The ``CommandsCache`` now has the ability to predict whether or not a
+  command must be run in the foreground using ``Popen`` or may use a
+  background thread and can use ``PopenThread``.
+* Callable aliases may now use the full gamut of functions signatures:
+  ``f()``, ``f(args)``,  ``f(args, stdin=None)``,
+  ``f(args, stdin=None, stdout=None)``, and `
+  ``f(args, stdin=None, stdout=None, stderr=None)``.
+* Uncaptured subprocesses now recieve a PTY file handle for stdout and
+  stderr.
+
+**Changed:**
+
+* The ``run_subproc()`` function has been replaced with a new implementation.
+* Piping between processes now uses OS pipes.
+* ``$XONSH_STORE_STDIN`` now uses ``os.pread()`` rather than ``tee`` and a new
+  file.
+
+**Deprecated:** None
+
+**Removed:**
+
+* ``CompletedCommand`` and ``HiddenCompletedCommand`` classes have been removed.
```

Comment:
It doesn't though, right, if you adjust your `$XONSH_PROC_FREQUENCY`... But I should add that to the news